tests/test_history.py

```diff
@@ -1751,3 +1751,99 @@ def test_skip_minutes_multiple(self, field):
                                     self.expected_values[asset][field][i],
                                     err_msg="sid={0} field={1} dt={2}".format(
                                         asset, field, minute))
+
+
+class DailyHistoryAggregatorTestCase(TestCase):
```

Comment:
I agree with your assessment that we should test the `DailyHistoryAggregator` directly.

My recommendation would be to change the existing tests which use the fixtures paradigm to use:

```
    def init_instance_fixtures(self):
        super(MinuteToDailyAggregationTestCase, self).init_instance_fixtures()
        # Set up a fresh data portal for each test, since order of calling
        # needs to be tested.
        self.equity_daily_aggregator = DailyHistoryAggregator(
            self.env.open_and_closes.market_open,
            self.bcolz_minute_bar_reader,
```